### DISCUSSION TITLE
FA/18C Hornet - Ext Light - Green

### DISCUSSION THREAD

**Discussion by Heli-Shed** [DCS-Skunkworks/dcs-bios#1023] [Q&A]
Hello.  Amazing DCS BIOS - I'm hooked with my home pit and I love your work!

Be grateful if you could take a look at the Address for the Right Green Nav light on the hornet.  it is essentially not working.

DcsBios::LED extPositionLightRight(0x7d46, 0x0800, 47);

I've checked the light with the apache - all good with that sketch but, it just wont work with the Hornet.  All other Ext lights it has to be said, are ok on the hornet for both sides.

Thank you.

Best

Tyger

**Comment by charliefoxtwo** [comment]
closing this as https://github.com/DCS-Skunkworks/dcs-bios/issues/1024 was opened
